pkg/rpc/server/testdata/test_verify.go

```diff
@@ -0,0 +1,15 @@
+package testdata
+
+import "github.com/nspcc-dev/neo-go/pkg/interop/runtime"
+
+func Main() int {
```

Comment:
Some comment will be useful. Are this hashes arbitrary?

Reply:
Yes, that's just random numbers.